src/thingset_bin.c

```diff
@@ -256,16 +264,19 @@ int ts_bin_process(struct ts_context *ts)
     return ts_bin_response(ts, TS_STATUS_BAD_REQUEST);
 }
 
-int ts_bin_fetch(struct ts_context *ts, const struct ts_data_node *parent, unsigned int pos_payload)
+/*
+* Remark: the parent object is currently still ignored. Any found data object is fetched.
```

Comment:
Can this become a Doxygen warning or note?

Reply:
Done.